lib/configure.js

```diff
@@ -260,7 +260,10 @@ function configure (gyp, argv, callback) {
       var candidates = ['include/node/node',
                         'out/Release/node',
                         'out/Debug/node',
-                        'node'].map(function(file) {
+                        'node',
+                        'out/Release/obj.target/libnode',
+                        'out/Debug/obj.target/libnode',
+                        'lib/libnode'].map(function(file) {
```

Comment:
Might be an idea to rewrite the comment just prior to this block as it's no longer AIX specific. We can probably drop the candidate locations from the comment as it's a duplicate of the code. Something like: 
```
// For AIX and z/OS we need to set up the path to the exports file
// which contains the symbols needed for linking. 
```

Also will the exports file ever be `include/node/node.x` on z/OS? If not, we could slightly speed up by using a different set of candidates per platform.

Reply:
Thanks @richardlau 
I will rewrite that comment.
Also the exports file on z/OS will never be `include/node/node.x`
So I will do as you suggested.